quora-service/src/main/java/com/upgrad/quora/service/business/AnswerService.java

```diff
@@ -0,0 +1,4 @@
+package com.upgrad.quora.service.business;
+
+public class AnswerService {
```

Comment:
it is better if we have multiple service files like - CreateAnserService, Delete AnswerService Etc. Am Merging the code. You can get it fixed in the next commits.

Reply:
Sure